include/bbcode.php

```diff
@@ -586,20 +586,26 @@ function bb_ShareAttributes($share, $simplehtml)
 			}
 			break;
 		default:
-			$text = trim($share[1]) . "\n";
-
-			$avatar = proxy_url($avatar, false, PROXY_SIZE_THUMB);
-
-			$tpl = get_markup_template('shared_content.tpl');
-			$text .= replace_macros($tpl, array(
-					'$profile' => $profile,
-					'$avatar' => $avatar,
-					'$author' => $author,
-					'$link' => $link,
-					'$posted' => $posted,
-					'$content' => trim($share[3])
-				)
-			);
+			// Transforms quoted tweets in rich attachments to avoid nested tweetsx
```

Comment:
Is tweetsx a typo or a new term (I am actually really confused with new terms being invented by everyone, so tweetsx might just be something I've never heard about before)

Reply:
It is a typo! 😄

Reply:
Do you want to fix it?

Reply:
Yes I will, I had friends over at home today which explains the lack of reaction.